website/util/permissions.py

```diff
@@ -7,7 +7,7 @@
 # NOTE: Ordered from most-restrictive to most permissive
 PERMISSIONS = [READ, WRITE, ADMIN]
 CREATOR_PERMISSIONS = [READ, WRITE, ADMIN]
-DEFAULT_CONTRIBUTOR_PERMISSIONS = [READ, WRITE]
+DEFAULT_CONTRIBUTOR_PERMISSIONS = [READ, WRITE, ADMIN]
```

Comment:
I don't think this is the change included in this pr. Please remove this change if it's not what you mean to do.